html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello Projects!</p>
+  <img src="https://miro.medium.com/max/1400/1*v3IHHAYzA9olTKiNksMUCQ.png" alt="Example of Code">
```

Comment:
Please use a relative url to the images provided in the starter files.

Reply:
<img width="912" alt="Screen Shot 2022-01-25 at 11 13 49 AM" src="https://user-images.githubusercontent.com/96929704/151043999-2c385f5c-5cbf-4b4c-8017-9874653a0201.png">